mm_100.html typos

benjub opened the pull request:
I fixed some typos in mm_100.html and standardize somewhat the format.

@david-a-wheeler, @jkingdon : for Line 25, there is a mention `revised 2007-01-20` but there is no trace of such a revision in https://us.metamath.org/mpeuni/sbth.html. Do you know if that revision really happened and what it was ? If not, I would simply remove this mention from the mm_100.html page.

For Line 30, the contribution date did not correspond to the one on the theorem webpage, so I assume it was a typo and retained the date on the theorem webpage (needless to say: I found this one by chance and did not check all ~100 dates!).

jkingdon commented:
> there is a mention `revised 2007-01-20` but there is no trace of such a revision in https://us.metamath.org/mpeuni/sbth.html. Do you know if that revision really happened and what it was ?

Well, here is the 2004 version of the proof which at first glance seems to be pretty similar to what we have today. Probably further digging could figure out what changed in 2007 (I'm sort of assuming something did, but also doubt it was major enough to need a mention in mm_100.html).

```
  ${
    $d x A $.  $d x B $.  $d x D $.  $d x f $.  $d x g $.
    sbthlem.1 $e |- A e. V $.
    sbthlem.2 $e |- D = { x |
                   ( x (_ A /\ ( g " ( B \ ( f " x ) ) ) (_ ( A \ x ) ) } $.
    $( Lemma for Schroeder-Bernstein Theorem. $)
    sbthlem1 $p |- U. D (_ ( A \ ( g " ( B \ ( f " U. D ) ) ) ) $=
      ( cuni cv cima cdif wss unissb wcel wa cleqabi wi ssconb biimprd exp
      difss sstr mpi syl5 pm2.43d imp sylbi elssuni imass2 sscon 3syl sstrd
      mprgbir ) DIZBFJZCEJZUOKZLZKZLZMAJZVAMADADVANVBDOZVBBUPCUQVBKZLZKZLZVAVCV
      BBMZVFBVBLZMZPZVBVGMZVKADHQVHVJVLVHVJVLVHVFBMZVJVLRZVJVHVMVNVHVMPVLVJVBVF
      BSTUAVJVIBMVMBVBUBVFVIBUCUDUEUFUGUHVCUSVEMZUTVFMVGVAMVCVBUOMVDURMVOVBDUIV
      BUOUQUJVDURCUKULUSVEUPUJUTVFBUKULUMUN $.
      $( [22-Mar-98] $)

    $( Lemma for Schroeder-Bernstein Theorem. $)
    sbthlem2 $p |- ( ran g (_ A ->
                     ( A \ ( g " ( B \ ( f " U. D ) ) ) ) (_ U. D ) $=
      ( cv crn wss cuni cima cdif wcel wa cab sbthlem1 imass2 ax-mp sscon wb
      wi imassrn sstr difss ssconb mpan2 syl mpbiri jctil ssexi wceq sseq1
      difeq2 sseq2d imaeq2 difeq2d 3syl bitrd bi2and elab sylibr eleq2i
      elssuni ) FIZJZBKZBVFCEIZDLZMZNZMZNZDOZVNVJKVHVNAIZBKZVFCVIVPMZNZMZBVPNZK
      ZPZAQZOZVOVHVNBKZVFCVIVNMZNZMZBVNNZKZPZWEVHWKWFVHWKVNBWINKZWIVMKZWMWHVLKZ
      WNVKWGKZWOVJVNKWPABCDEFGHRVJVNVISTVKWGCUATWHVLVFSTWIVMBUATVHWIBKZWKWMUBZW
      IVGKVHWQUCVFWHUDWIVGBUETWQWFWRBVMUFZWIVNBUGUHUIUJWSUKWCWLAVNVNBGWSULVPVNU
      MZVQWFWBWKVPVNBUNWTWBVTWJKZWKWTWAWJVTVPVNBUOUPWTVSWHUMVTWIUMXAWKUBWTVRWGC
      VPVNVIUQURVSWHVFUQVTWIWJUNUSUTVAVBVCDWDVNHVDVCVNDVEUI $.
      $( [22-Mar-98] $)

    $( Lemma for Schroeder-Bernstein Theorem. $)
    sbthlem3 $p |- ( ran g (_ A ->
                  ( g " ( B \ ( f " U. D ) ) ) = ( A \ U. D ) ) $=
      ( cv crn wss cuni cdif cima wa wceq sbthlem2 sbthlem1 jctil eqss sylibr
      difeq2d wi imassrn sstr ax-mp dfss4 sylib eqtr2d ) FIZJZBKZBDLZMBBUJCEIUM
      NMZNZMZMZUOULUMUPBULUMUPKZUPUMKZOUMUPPULUSURABCDEFGHQABCDEFGHRSUMUPTUAUBU
      LUOBKZUQUOPUOUKKULUTUCUJUNUDUOUKBUEUFUOBUGUHUI $.
      $( [22-Mar-98] $)

    $( Lemma for Schroeder-Bernstein Theorem. $)
    sbthlem4 $p |- ( ( ( dom g = B /\ ran g (_ A ) /\ Fun `' g ) ->
                  ( `' g " ( A \ U. D ) ) = ( B \ ( f " U. D ) ) ) $=
      ( cv cdm wceq crn wss wa ccnv wfun cuni cima cdif cres difss sseq2
      mpbiri ssdmres sylib dfdm4 syl5reqr funcnvres sbthlem3 reseq2 syl
      sylan9eqr rneqd sylan9eq anassrs df-ima syl6reqr ) FIZJZCKZURLBMZNUROZPZN
      CEIDQZRZSZVBBVDSZTZLZVBVGRUTVAVCVFVIKUTVAVCNZVFURVFTZOZLZVIUTVKJZVFVMUTVF
      USMZVNVFKUTVOVFCMCVEUAUSCVFUBUCVFURUDUEVKUFUGVJVLVHVCVAVLVBURVFRZTZVHVFUR
      UHVAVPVGKVQVHKABCDEFGHUIVPVGVBUJUKULUMUNUOVBVGUPUQ $.
      $( [27-Mar-98] $)

    ${
      $d x H $.
      sbthlem.3 $e |- H = ( ( f |` U. D ) u. ( `' g |` ( A \ U. D ) ) ) $.

      $( Lemma for Schroeder-Bernstein Theorem. $)
      sbthlem5 $p |- ( ( dom f = A /\ ran g (_ A ) -> dom H = A ) $=
        ( cv cdm wceq crn wss wa cuni cdif cun cin cima sbthlem1 difss sstri
        sseq2 mpbiri dfss sylib uneq1d imassrn sbthlem3 sseq1d mpbii uneq2d
        sylan9eq cres ccnv dmeqi dmun dmres df-rn cleqcomi ineq2i eqtr uneq12i
        3eqtr syl6reqr ssundif mpbi syl6eq ) EKZLZBMZFKZNZBOZPZGLZDQZBVSRZSZBVQ
        WAVSVLTZVTVOTZSZVRVMVPWAWBVTSWDVMVSWBVTVMVSVLOZVSWBMVMWEVSBOZVSBVNCVKVS
        UARZUAZRBABCDEFHIUBBWHUCUDZVLBVSUEUFVSVLUGUHUIVPVTWCWBVPVTVOOZVTWCMVPWH
        VOOWJVNWGUJVPWHVTVOABCDEFHIUKULUMVTVOUGUHUNUOVRVKVSUPZVNUQZVTUPZSZLWKLZ
        WMLZSWDGWNJURWKWMUSWOWBWPWCVKVSUTWPVTWLLZTWCWLVTUTWQVOVTVOWQVNVAVBVCVDV
        EVFVGWFWABMWIVSBVHVIVJ $.
        $( [22-Mar-98] $)

      $( Lemma for Schroeder-Bernstein Theorem. $)
      sbthlem6 $p |- ( ( ran f (_ B /\
               ( ( dom g = B /\ ran g (_ A ) /\ Fun `' g ) ) -> ran H = B ) $=
        ( cv cdm wceq crn wss wa ccnv wfun cuni cima cdif cun cres sbthlem4
        df-ima syl5reqr uneq2d rnun rneqi uneq1i 3eqtr4 syl6reqr wi imassrn
        sstr ax-mp ssundif sylib sylan9eqr ) FKZLCMUTNBOPUTQZRPZEKZNZCOZGNZVCDS
        ZTZCVHUAZUBZCVBVJVHVABVGUAZUCZNZUBZVFVBVIVMVHVBVAVKTVIVMABCDEFHIUDVAVKU
        EUFUGVCVGUCZVLUBZNVONZVMUBVFVNVOVLUHGVPJUIVHVQVMVCVGUEUJUKULVEVHCOZVJCM
        VHVDOVEVRUMVCVGUNVHVDCUOUPVHCUQURUS $.
        $( [27-Mar-98] $)

      $( Lemma for Schroeder-Bernstein Theorem. $)
      sbthlem7 $p |- ( ( Fun f /\ Fun `' g ) -> Fun H ) $=
        ( cv wfun ccnv wa cuni cres cdif cun cdm cin c0 wceq wss dmres inss1
        eqsstr ssrin ax-mp sslin sstri indif0 sseqtr ss0 funun mpan2 funres
        syl2an wb funeq sylibr ) EKZLZFKMZLZNVADOZPZVCBVEQZPZRZLZGLZVFLZVHLZVJV
        BVDVLVMNVFSZVHSZTZUAUBZVJVPUAUCVQVPVEVGTZUAVPVEVOTZVRVNVEUCVPVSUCVNVEVA
        SZTVEVAVEUDVEVTUEUFVNVEVOUGUHVOVGUCVSVRUCVOVGVCSZTVGVCVGUDVGWAUEUFVOVGV
        EUIUHUJVEBUKULVPUMUHVFVHUNUOVEVAUPVGVCUPUQGVIUBVKVJURJGVIUSUHUT $.
        $( [27-Mar-98] $)

      $( Lemma for Schroeder-Bernstein Theorem. $)
      sbthlem8 $p |- ( ( Fun `' f /\ ( ( ( Fun g /\ dom g = B ) /\ ran g (_ A )
                     /\ Fun `' g ) ) -> Fun `' H ) $=
        ( cv ccnv wfun cdm wceq wa crn wss cuni cres cdif cun cin c0 funun
        funres11 funcnvcnv syl adantr ad2antll im2an cima sbthlem4 df-ima
        df-rn eqtr syl5eqr eqtr2 jctil ineq12 indif0 syl6eq pm3.27 imran sylan
        adantl sylan13 wb cnveq ax-mp cnvun funeq sylibr ) EKZLMZFKZMZVPNCOZPZV
        PQBRZPZVPLZMZPZPZVNDSZTZLZWBBWFUAZTZLZUBZMZGLZMZWHMZWKMZPWHNZWKNZUCZUDO
        ZWMWEWHWKUEVOWPWDWQWFVNUFVSWQVTWCVQWQVRVQWBLMWQVPUGWIWBUFUHUIUJUKWDXAVO
        VRVTPZWCXAWAXBWCPZWTVNWFULZCXDUAZUCZUDXCWRXDOZWSXEOZPWTXFOXCXHXGXCWBWIU
        LZXEWSABCDEFHIUMXIWJQWSWBWIUNWJUOUPUQXDWGQWRVNWFUNWGUOURUSWRXDWSXEUTUHX
        DCVAVBVSVRVTVQVRVCVDVEVFVGWNWLOWOWMVHWNWGWJUBZLZWLGXJOWNXKOJGXJVIVJWGWJ
        VKUPWNWLVLVJVM $.
        $( [27-Mar-98] $)

      $( Lemma for Schroeder-Bernstein Theorem. $)
      sbthlem9 $p |- ( ( f : A -1-1-> B /\ g : B -1-1-> A ) ->
                    H : A -1-1-onto-> B ) $=
        ( cv wfun cdm wceq wa crn wss ccnv wf1 wf1o sbthlem7 sbthlem5 adantrl
        im2an an41r3s adantlr sbthlem8 adantll sbthlem6 df-rn syl5eqr pm3.27
        imran sylan5 jca wf df-f1 wfn df-f df-fn biran bitr bi2an f1o4 3imtr4
        ) EKZLZVFMBNZOZVFPCQZOZVFRLZOZFKZLZVNMCNZOZVNPBQZOZVNRLZOZOZGLZGMBNZOZG
        RZLZWFMZCNZOZOZBCVFSZCBVNSZOBCGTZWBWEWJVKWAWEVLVIWAWEVJVGVTVHVSWEVGVTOW
        CVHVSOWDABCDEFGHIJUAVHVRWDVQABCDEFGHIJUBUCUDUEUFUFWBWGWIVLWAWGVKABCDEFG
        HIJUGUHVKWAWIVLVJWAWIVIVJVPVROZVTOZWIWAVJWPOGPCWHABCDEFGHIJUIGUJUKVSWOV
        TVQVPVRVOVPULUMUMUNUHUFUOUOWLVMWMWAWLBCVFUPZVLOVMBCVFUQWQVKVLWQVFBURZVJ
        OVKBCVFUSWRVIVJVFBUTVAVBVAVBWMCBVNUPZVTOWACBVNUQWSVSVTWSVNCURZVROVSCBVN
        USWTVQVRVNCUTVAVBVAVBVCWNGBURZWFCURZOWKBCGVDXAWEXBWJGBUTWFCUTVCVBVE $.
        $( [28-Mar-98] $)

      ${
        $d f g A $.  $d f g B $.
        sbthlem.4 $e |- B e. V $.
        $( Lemma for Schroeder-Bernstein Theorem. $)
        sbthlem10 $p |- ( ( A ~<_ B /\ B ~<_ A ) -> A ~~ B ) $=
          ( cdom wbr wa cv wf1 wex cen brdom bi2an eeanv bitr4 wf1o sbthlem9
          f1oen syl 19.23aivv sylbi ) BCLMZCBLMZNZBCEOPZCBFOPZNZFQEQZBCRMZUKULE
          QZUMFQZNUOUIUQUJURBCEKSCBFHSTULUMEFUAUBUNUPEFUNBCGUCUPABCDEFGHIJUDBCG
          HUEUFUGUH $.
          $( [28-Mar-98] $)
      $}
    $}

  $}

  ${
    $d x y z w f g A $.  $d x y z w f g B $.  $( ` x y z w f g ` are dummy. $)
    $( Schroeder-Bernstein Theorem.  Theorem 18 of [Suppes] p. 95.  This theorem
       states that if set ` A ` is smaller (has lower cardinality) than ` B `
       and vice-versa, then ` A ` and ` B ` are equinumerous (have the same
       cardinality).  The interesting thing is that this can be proved without
       invoking the Axiom of Choice, as we do here, but the proof as you can
       see is quite difficult.  (The theorem can be proved more easily if we
       allow AC.)  The main proof consists of lemmas ~ sbthlem1 through
       ~ sbthlem10 ; this final piece mainly changes bound variables to
       eliminate the hypotheses of ~ sbthlem10 . $)
    sbth $p |- ( ( A ~<_ B /\ B ~<_ A ) -> A ~~ B ) $=
      ( vz vw vx vy vg vf cdom wbr wa cen cvv wcel wi cv wceq breq1 breq2
      bi2and bi2imd wss cima cdif cab cuni cres ccnv cun visset weq sseq1 wb
      imaeq2 difeq2d 3syl difeq2 sseq2d bitrd cbvabv cleqid sbthlem10 vtocl2g
      reldom brrelexi syl2an pm2.43i ) ABIJZBAIJZKZABLJZAMNBMNVJVKOZVHVICPZDPZI
      JZVNVMIJZKZVMVNLJZOAVNIJZVNAIJZKZAVNLJZOVLCDABMMVMAQZVQWAVRWBWCVOVSVPVTVM
      AVNIRVMAVNISTVMAVNLRUAVNBQZWAVJWBVKWDVSVHVTVIVNBAISVNBAIRTVNBALSUAEVMVNFP
      ZVMUBZGPZVNHPZWEUCZUDZUCZVMWEUDZUBZKZFUEZHGWHWOUFZUGWGUHVMWPUDUGUIZCUJWNE
      PZVMUBZWGVNWHWRUCZUDZUCZVMWRUDZUBZKFEFEUKZWFWSWMXDWEWRVMULXEWMXBWLUBZXDXE
      WJXAQWKXBQWMXFUMXEWIWTVNWEWRWHUNUOWJXAWGUNWKXBWLULUPXEWLXCXBWEWRVMUQURUST
      UTWQVADUJVBVCABIVDVEBAIVDVEVFVG $.
      $( [8-Jun-98] $)
  $}
```

jkingdon commented:
> For Line 30, the contribution date did not correspond to the one on the theorem webpage, so I assume it was a typo and retained the date on the theorem webpage

I guess I'd keep the 2017 date. At least, that was when it was merged in - https://github.com/metamath/set.mm/pull/112

benjub commented:
Thanks @jkingdon. I removed that mention of the revision in a commit.
I reverted the date change for Line 30 (theorem ~ballotth). I think we should have the same date in set.mm and in mm_100.html. Maybe @tirix can confirm, but my guess is that the earlier date corresponds to when Thierry actually proved it on a local branch, while the later date is as you said the date of the merge.

So: should I put the later date for both ? or the earlier date for both ?

david-a-wheeler commented:
We usually use the earlier date ("date proved"), not "date merged into set.mm and/or put in main". Historically (especially before we used git) things could languish for a while before they got merged.

benjub commented:
Ok, so I did put the original contribution date in the mm_100.html page, as I originally did in the first commit of this PR and as @david-a-wheeler wrote aboe.  I hope it's ok with @jkingdon.

It is true that in some sense we trust @tirix that it was done on that date indeed.  This would be important if for instance someone else had published a similar result in between these two dates. If anyone voices doubts on the anteriority of a result, then the earliest date one can claim has to be of a PR to a public repository, or anything similar which is independently verifiable.  Such arguments have not happened in metamath to my knowledge, and hopefully they will not.

jkingdon commented:
> so I did put the original contribution date in the mm_100.html page, as I originally did in the first commit of this PR and as @david-a-wheeler wrote aboe. I hope it's ok with @jkingdon.

I think I've pretty much lost interest in the topic, so I guess that counts as "yes it is OK".
